ome/resource_discovery_schema.go

```diff
@@ -0,0 +1,469 @@
+package ome
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/listvalidator"
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// DiscoveryJobSchema for discovery job schema.
+func DiscoveryJobSchema() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+
+		"discovery_job_id": schema.Int64Attribute{
+			MarkdownDescription: "ID of the discovery configuration group",
+			Description:         "ID of the discovery configuration group",
+			Computed:            true,
+		},
+
+		"discovery_job_name": schema.StringAttribute{
+			MarkdownDescription: "Name of the discovery configuration job",
+			Description:         "Name of the discovery configuration job",
+			Required:            true,
+			Validators: []validator.String{
+				stringvalidator.LengthAtLeast(1),
+			},
+		},
+
+		"email_recipient": schema.StringAttribute{
+			MarkdownDescription: `
+				- Enter the email address to which notifications are to be sent about the discovery job status.
+				- Configure the SMTP settings to allow sending notifications to an email address.`,
+			Description: `
+				- Enter the email address to which notifications are to be sent about the discovery job status.
+				- Configure the SMTP settings to allow sending notifications to an email address.`,
+			Optional: true,
+			Computed: true,
```

Comment:
Please confirm if this is supposed to be computed as well.

Reply:
remove the computed as true from the attribute